src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,139 @@
-export const TodoList = () => {};
+import React, { useState } from 'react';
+import usersFromServer from '../../api/users';
+import { Todo } from '../types/Todo';
+import { getUserById } from '../../services/user';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  onAdd: (todo: Todo) => void;
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({
+  onAdd,
+  todos,
+}) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasUserIdError, setHasUserIdError] = useState(false);
+
+  const areRequiredFieldsFilled = () => {
+    return (
+      title.trim() === '' && userId === 0
```

Comment:
```suggestion
      title.trim() === '' || userId === 0
```
<img width="340" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/4f824fea-af4f-44d4-9cfe-6e62936c7c9c">
<img width="282" alt="image" src="https://github.com/mate-academy/react_add-todo-form/assets/70220667/39cf6831-d580-4efd-beb9-bb42330f8d9f">